frappe/website/web_template/subnav/subnav.json

```diff
@@ -0,0 +1,74 @@
+{
+ "creation": "2020-05-15 17:53:36.526784",
+ "docstatus": 0,
+ "doctype": "Web Template",
+ "fields": [
+  {
+   "fieldname": "title",
+   "fieldtype": "Data",
+   "label": "Title",
+   "reqd": 0
+  },
+  {
+   "fieldname": "logo",
+   "fieldtype": "Attach Image",
+   "label": "Logo",
+   "reqd": 0
+  },
+  {
+   "fieldname": "label_1",
+   "fieldtype": "Data",
+   "label": "Label 1",
+   "reqd": 0
+  },
+  {
+   "fieldname": "url_1",
+   "fieldtype": "Data",
+   "label": "URL 1",
+   "reqd": 0
+  },
+  {
+   "fieldname": "label_2",
+   "fieldtype": "Data",
+   "label": "Label 2",
+   "reqd": 0
+  },
+  {
+   "fieldname": "url_2",
+   "fieldtype": "Data",
+   "label": "URL 2",
+   "reqd": 0
+  },
+  {
+   "fieldname": "label_3",
+   "fieldtype": "Data",
+   "label": "Label 3",
+   "reqd": 0
+  },
+  {
+   "fieldname": "url_3",
+   "fieldtype": "Data",
+   "label": "URL 3",
+   "reqd": 0
+  },
+  {
+   "fieldname": "label_4",
+   "fieldtype": "Data",
+   "label": "Label 4",
+   "reqd": 0
+  },
+  {
+   "fieldname": "url_4",
+   "fieldtype": "Data",
+   "label": "URL 4",
+   "reqd": 0
+  }
+ ],
+ "idx": 0,
+ "modified": "2020-05-15 17:54:52.253976",
+ "modified_by": "Administrator",
+ "name": "Subnav",
```

Comment:
This doesn't have anything to do with Subnav. Just call it `Navbar` and then add the ability to disable Navbar from Website Settings, so this can be used instead.

Reply:
> disable Navbar from Website Settings

In that case it will disable navbar across the website, which is not desired